src/renderer/src/components/trigger/TriggersSlice.ts

```
@@ -25,11 +27,11 @@ const initialState = triggersAdapter.getInitialState({
 // thunk functions
 export const fetchTrigger = createAsyncThunk(
   'triggers/fetchTrigger',
-  async (query) => {
+  async ({ email }: { email: string }) => {
     const response = await IpcService.send<{ trigger: any }>(
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)